src/frontend/apps/standalone_site/src/features/Authentication/components/BaseAuthenticationPage.tsx

```diff
@@ -65,12 +76,67 @@ const ResponsiveBox = ({ children }: PropsWithChildren) => {
         background={backgroundImage('center', isViewWidth ? '45vw' : '80vh')}
         style={{ flex: '1' }}
       >
-        <LogoIcon
-          color="white"
-          width="80%"
-          height="30%"
-          aria-label={intl.formatMessage(messages.altLogo)}
-        />
+        {siteConfig.isDefaultSite ? (
+          <LogoIcon
+            color="white"
+            width="80%"
+            height="30%"
+            aria-label={intl.formatMessage(messages.altLogo)}
+          />
+        ) : (
+          <Box
+            width="large"
+            margin="auto"
+            pad={{ vertical: 'small', horizontal: 'medium' }}
+            align="center"
+          >
+            <ReactMarkdown
+              rehypePlugins={[rehypeRaw]}
+              components={{
```

Comment:
I don't know if it is the best way, because the `loginMarkdown` will be highly dependent of this code. The code could help a bit the design but with things very generic. As it is now if a user will want to create his own login page it will be complicated, he will probably have to come here as well to modify or add some code.
IMO the markdown page should design itself.

Reply:
Could you try to remove the `components` prop and add that in the admin dashboard ? Tell me what you think about.
```
<div style="height:65vh;display:flex;flex-direction:column">
<H1 style="text-align: center;color:white;line-height: initial;">Le portail national <br/> des classes virtuelles  <br/> <span style="font-weight:lighter;">de l'enseignement supérieur</span></H1 >

<div style="display: flex;gap: 1rem;align-items: center;justify-content: center;"><img src="http://localhost:3001/logos/fun.png" width="100"><img  width="150" src="http://localhost:3001/logos/bbb.png"></div>

<div style="margin-top:auto;padding:0.5rem 1rem;background-color:#0556BF;border-radius:5rem;display: flex;gap: 1rem;align-items: center;justify-content: center;">
<p style="color:white;font-size:12px;text-align: right;line-height: 14px;">Financé<br/>par le plan<br/>de relance</p>
<img src="http://localhost:3001/logos/ministere.png" width="100"><img  width="55" src="http://localhost:3001/logos/france-relance.png"><img  width="100" src="http://localhost:3001/logos/europe.png">
</div></div>
```

Reply:
I'm not sure to get your point…
Isn't tho whole point of markdown to separate content and form ?
I mean, having to edit this kind of html with inlined css will be a pain, IMO.
Maybe we should consider to use some django template if we don't want to use…